content/applications/finance/fiscal_localizations/united_kingdom.rst

```diff
@@ -81,6 +81,8 @@ The **UK - Accounting Reports** module enables you to comply with the `HM Revenu
    it through Odoo, as Odoo only retrieves open bonds from the last three months. Your submission
    has to be done manually by contacting HMRC.
 
+.. _content/applications/finance/fiscal_localizations/register-company-to-hmrc:
```

Comment:
```suggestion
.. _uk_localization/hmrc-registration:
```
While there are no 'naming conventions' when creating [custom anchors](https://www.odoo.com/documentation/16.0/contributing/documentation/rst_cheat_sheet.html#custom-anchors), it isn't necessary to write down the full path. Generally, we keep them short while keeping them distinct enough to avoid any (future) potential duplicates. Feel free to adapt my suggestion as you see fit.